x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/synthetics/kpi_over_time_config.ts

```diff
@@ -94,6 +94,7 @@ export function getSyntheticsKPIConfig({ dataView }: ConfigProps): SeriesConfig
         id: 'monitor_availability',
         columnType: FORMULA_COLUMN,
         formula: "1- (count(kql='summary.down > 0') / count())",
```

Comment:
```suggestion
        formula: "1- (count(kql='summary.down > 0') / count(kql='summary: 0'))",
```

maybe do this, needs less changes